rise-data.html

```diff
@@ -120,7 +120,7 @@
       _getCacheUrl: function( key, method ) {
         var url = this._baseCacheUrl + "/" + this.endpoint;
 
-        if ( method === "PUT" || method === "DELETE" || method === "GET" ) {
+        if ( method === "DELETE" || method === "GET" ) {
```

Comment:
So if an invalid method is requested it's going to continue as though the request was made with no key?  Then what?  Why aren't we returning a 405?